src/app/counter-analytics/day-of-week-histogram/day-of-week-histogram.page.component.html

```diff
@@ -0,0 +1,15 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-back-button icon="arrow-back" defaultHref="/counter-analytics/{{ counter.id }}"></ion-back-button>
```

Comment:
when do we use the `close` and when do we use the `arrow-back` icon?
I do like how you have done this now (close on the overview and arrow back here) but do we have a concept for this?..

Reply:
Personally I'm not used to the `close` icon and I found it odd to press it twice to return to the main screen. Maybe we can use `close` if some data is potentially discarded and `arrow-back` otherwise?

Reply:
I do like this rule. We should stick to that.

Reply:
Changed all buttons to `arrow-back` in the analytics menu.